docs/conf.py

```diff
@@ -89,17 +89,17 @@
 
 # General information about the project.
 project = 'Procrustes'
-copyright = '2018, Ayers Lab'
+copyright = '2020, Ayers Lab'
 author = 'Ayers Lab'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '1.0.0'
+version = '0.0.1'
 # The full version, including alpha/beta/rc tags.
-release = '1.0.0'
+release = '0.0.1alpha'
```

Comment:
release format should be like 0.0.0-alpha, with a hyphen

Reply:
Fixed in 6687ea2. Thank you for the careful inspection.